src/client_side_reply.cc

```diff
@@ -2255,6 +2255,21 @@ clientReplyContext::sendMoreData (StoreIOBuffer result)
     return;
 }
 
+// whether collapsed forwarding is on and allowed
+// for current transaction
```

Comment:
Here is a slightly better one-liner:

```
/// whether becoming a CF master or slave is allowed (partial check!)
```

or, if you think a negative statement is better in this special case:

```
/// whether becoming a CF master or slave is not prohibited
```